packages/cohort_sdk_js/src/initiator/mod.rs

```diff
@@ -67,12 +67,73 @@ impl From<JsInitiatorConfig> for Config {
     }
 }
 
+// #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+// #[serde(rename_all = "camelCase")]
+#[napi(object)]
+pub struct JSKafkaAction {
```

Comment:
Minor, not a blocker for PR merge. Just most of the data structures use "Js" prefix (here we seems to use "JS")

Reply:
Oops, that should be `Js` I will update that soon. Thanks for spotting it